client/js/reducers/progress.js

```diff
@@ -8,7 +8,7 @@ const initialState = Immutable.fromJS({
   isStarted: false,
   currentItemIndex: 0,
   selectedAnswerId: '',
-  answerMessageIndex: '',
+  answerMessageIndex: [],
```

Comment:
Because we need to have an index for each question, or at least that was what made sense to me.  Progress holds information about all of the questions, for instance question[0] could be 1 (CORRECT_RESPONSE) and question[1] could have 2 (INCORRECT_RESPONSE).